docs/handbook/text-anchors.rst

```diff
@@ -0,0 +1,140 @@
+  :alt: Horizontal text
+  :align: center
+
+.. image:: ../resources/anchor_vertical.svg
+  :alt: Vertical text
+  :align: center
+
+Horizontal anchor alignment
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``l`` --- left
+    Anchor is to the left of the text.
+
+    For *horizontal* text this is the origin of the first glyph, as shown in the `FreeType tutorial`_.
+
+``m`` --- middle
+    Anchor is horizontally centered with the text.
+
+    For *vertical* text it is recommended to use ``s`` (baseline) alignment instead,
+    as it does not change based on the specific glyphs of the given text.
+
+``r`` --- right
+    Anchor is to the right of the text.
+
+    For *horizontal* text this is the advanced origin of the last glyph, as shown in the `FreeType tutorial`_.
+
+``s`` --- baseline *(vertical text only)*
+    Anchor is at the baseline (middle) of the text. The exact alignment depends on the font.
+
+    For *vertical* text this is the recommended alignment,
+    as it does not change based on the specific glyphs of the given text
+    (see image for vertical text above).
+
+Vertical anchor alignment
+^^^^^^^^^^^^^^^^^^^^^^^^^
+
+``a`` --- ascender / top *(horizontal text only)*
+    Anchor is at the ascender line (top) of the first line of text, as defined by the font.
+
+    See `Font metrics on Wikipedia`_ for more information.
+
+``t`` --- top *(single-line text only)*
+    Anchor is at the top of the text.
+
+    For *vertical* text this is the origin of the first glyph, as shown in the `FreeType tutorial`_.
+
+    For *horizontal* text it is recommended to use ``a`` (ascender) alignment instead,
+    as it does not change based on the specific glyphs of the given text.
+
+``m`` --- middle
+    Anchor is vertically centered with the text.
+
+    For *horizontal* text this is the midpoint of the first ascender line and the last descender line.
+
+``s`` --- baseline *(horizontal text only)*
+    Anchor is at the baseline (bottom) of the first line of text, only descenders extend below the anchor.
+
+    See `Font metrics on Wikipedia`_ for more information.
+
+``b`` --- bottom *(single-line text only)*
+    Anchor is at the bottom of the text.
+
+    For *vertical* text this is the advanced origin of the last glyph, as shown in the `FreeType tutorial`_.
+
+    For *horizontal* text it is recommended to use ``d`` (descender) alignment instead,
+    as it does not change based on the specific glyphs of the given text.
+
+``d`` --- descender / bottom *(horizontal text only)*
+    Anchor is at the descender line (bottom) of the last line of text, as defined by the font.
+
+    See `Font metrics on Wikipedia`_ for more information.
+
+Examples
+^^^^^^^^
+
+The following image shows several examples of anchors for horizontal text.
+In each section the ``xy`` parameter was set to the center shown by the intersection
+of the two lines.
+
+.. comment: Image generated with ../example/anchors.py
+
+.. image:: ../example/anchors.png
```

Comment:
Minor suggestion:

This image may be a bit easier to read if the "Sample" text was replaced with "anchor=ma" etc. That is, move "anchor=ma" etc. to the position where "Sample" is now.

Then there's no need to keep glancing back and forth between each pair.

Reply:
Good suggestion, but "anchor=ma" doesn't contain any descenders, so "md" and "mb" would look identical. I changed it to "ma example" instead.